### ISSUE TITLE
Add iOS7.1 target to the Travis compilation matrix

### ISSUE THREAD

**Issue by AliSoftware** [AliSoftware/OHHTTPStubs#87]
We should improve that `.travis.yml` configuration, because I think it could be interesting to run the Unit tests both with destination `OS=7.1` and `OS=8.1`

Especially, some issues did arise in the past due to modifications in iOS8, and I had to fix them by ensuring both that it adapts to the new behavior of iOS8… while still working on iOS7.
_That was the case for #65 and #82 (issues both introduced by changes in iOS8)_

So we should ensure both that it works on iOS8 while it does not introduce regressions on iOS7.

For reference, this consists of adding some lines to `.travis.yml` and varying the configurations used in the `-destination` parameter, like: `-destination 'platform=iOS Simulator,name=iPhone 6,OS=…'`

**Comment by AliSoftware** [comment]
For reference we may probably use the Travis build matrix feature for that: http://docs.travis-ci.com/user/build-configuration/#The-Build-Matrix

**Comment by AliSoftware** [comment]
Done. Travis now shows a nice build matrix to show the status of each configuration separately (so if one fails we can see which one and investigate in its build log only, without being lost in a too long, unique console log)

![Travis-Matrix-Capture](https://cloud.githubusercontent.com/assets/216089/6427165/cbcad9bc-bf78-11e4-96f6-bccd7b5341b2.jpg)

**Comment by AliSoftware** [comment]
**[Update]** It appears that Travis recently dropped the iOS7 Runtime Simulator on their OS X VMs. Or maybe we got lucky when building the last time and were attributed a VM that had it installed, but it does not seem to be the case on every Travis OSX VMs.

I emailed their support to ask them to add that iOS7 Runtime to all their OS X VMs, and disabled the compilation on iOS7.1 in the meantime. Wait & see.

_I'm reopening this issue to keep track of how things evolve with them and see if we can add it back in the future or if we will have to drop it._
